### ISSUE TITLE
ERROR deprecated: .Site.IsServer was deprecated in Hugo v0.120.0 and will be removed in Hugo 0.133.0. Use hugo.IsServer instead.

### ISSUE THREAD

**Issue by KHannibal** [Lednerb/bilberry-hugo-theme#655]
Using hugo v.0.132.1+extended and the last version of bilberry (4.8.0), I am unable to build the site because of the aforementioned error.

```
hugo: downloading modules …
go: downloading github.com/Lednerb/bilberry-hugo-theme/v4 v4.8.0
go: downloading github.com/Lednerb/bilberry-hugo-theme v1.0.1
hugo: collected modules in 5623 ms
Start building sites … 
hugo v0.132.1+extended darwin/arm64 BuildDate=2024-08-13T10:10:10Z VendorInfo=brew

ERROR deprecated: .Site.IsServer was deprecated in Hugo v0.120.0 and will be removed in Hugo 0.133.0. Use hugo.IsServer instead.
Total in 7068 ms
Error: error building site: logged 1 error(s)
```

`.Site.IsServer` is used in [v4/layouts/partials/js.html](https://github.com/Lednerb/bilberry-hugo-theme/blob/60203bb5d0513a8c113bc435179aba2a09398fa1/v4/layouts/partials/js.html#L30), [v4/layouts/partials/css.html](https://github.com/Lednerb/bilberry-hugo-theme/blob/60203bb5d0513a8c113bc435179aba2a09398fa1/v4/layouts/partials/css.html#L4) and [v4/layouts/partials/check-theme-prefer.html](https://github.com/Lednerb/bilberry-hugo-theme/blob/60203bb5d0513a8c113bc435179aba2a09398fa1/v4/layouts/partials/check-theme-prefer.html#L2).

**Comment by igor-baiborodine** [comment]
@KHannibal, as a temporary workaround, downgrade your Hugo version to `v0.125.7`. This issue will be fixed in `v4.9.0'.

**Comment by igor-baiborodine** [comment]
Fixed in [v4.9.0](https://github.com/Lednerb/bilberry-hugo-theme/releases/tag/v4.9.0) release.
